docs/source/azure/configuration/authentication.rst

```diff
@@ -112,6 +112,32 @@ You must set `AZURE_SUBSCRIPTION_ID` as with the other authentication types.
     AZURE_SUBSCRIPTION_ID=subscriptionId
     AZURE_CLIENT_ID=clientId
 
+Azure Key Vault Integration
+---------------------------
+
+If you run Custodian inside Azure VM, AKS, ACI or Azure Function you can leverage Azure Key Vault to store
+Service Principal credentials. You can store json-formatted authentication file as a Key Vault secret.
```

Comment:
(minor) `You can store the json-formatted authentication file as a Key Vault secret.`